controllers/triggerservice/triggerservice_controller.go

```diff
@@ -183,6 +208,44 @@ func (r *Reconciler) createRoles(ctx context.Context, ts *standardv1alpha1.Trigg
 	return nil
 }
 
+func (r *Reconciler) createService(ctx context.Context, ts *standardv1alpha1.TriggerService, v cue.Value) error {
+	needCreateService, err := v.LookupPath(cue.ParsePath("parameter.createService")).Bool()
+	if err != nil {
+		return err
+	}
+
+	if !needCreateService {
```

Comment:
If the `needCreateService` is false, then we'll try to delete the existing service? I think it is a dangerous behavior and we shouldn't clean the service for user automatically.

Reply:
should we add another flag to clean the service when user want to clean service？